base_control/Odometry.h

```diff
@@ -59,9 +59,9 @@ class Odometry{
         
         // Suscriber.
         ros::Subscriber<geometry_msgs::Twist, Odometry> velocity_subscriber_;
-        static constexpr double kLinearXMaxVelocity = 0.39;
-        static constexpr double kLinearYMaxVelocity = 0.39;
-        static constexpr double kAngularZMaxVelocity = 0.39;
+        static constexpr double kLinearXMaxVelocity = 0.19;
+        static constexpr double kLinearYMaxVelocity = 0.19;
+        static constexpr double kAngularZMaxVelocity = 0.19;
```

Comment:
Good job! please add the images you sent in whatsapp to this PR as a comment

Reply:
Done!